Dockerfile

```diff
@@ -33,8 +32,11 @@ RUN chown -R hypothesis:hypothesis /etc/nginx/conf.d /var/log/nginx /var/lib/ngi
 COPY requirements/prod.txt ./
 
 # Install build deps, build, and then clean up.
-RUN pip install --no-cache-dir -U pip \
-  && pip install --no-cache-dir -r prod.txt
+RUN apt-get install --yes gcc \
+  && pip install --no-cache-dir -U pip \
+  && pip install --no-cache-dir -r prod.txt \
+  && apt-get remove --yes gcc \
+  && apt-get autoremove --yes
```

Comment:
We only need `gcc` during the `pip install` command, we don't need to leave it in the container, so I installed it just before it's needed and then removed it again.

Why do we need `gcc` though? It's because our `Dockerfile` is building `psycopg2` from source. I wonder if there's a different fix here where we could get `pip` to install ` pre-built `psycopg2` binary?

Reply:
There's https://pypi.org/project/psycopg2-binary/

but the docs there say:

```
You can also obtain a stand-alone package, not requiring a compiler or
external libraries, by installing the `psycopg2-binary`_ package from PyPI::

    $ pip install psycopg2-binary

The binary package is a practical choice for development and testing but in
production it is advised to use the package built from sources.
```